components/AvalancheForecastZonePolygon.tsx

```diff
@@ -20,25 +19,19 @@ const toLatLng = (item: number[]): LatLng => {
   return {longitude: item[0], latitude: item[1]};
 };
 
+export const toLatLngList = (geometry: FeatureComponent): LatLng[] => {
+  return coordinateList(geometry).map(toLatLng);
+};
+
 export interface AvalancheForecastZonePolygonProps {
   zone: MapViewZone;
-  setRegionBounds: React.Dispatch<React.SetStateAction<RegionBounds>>;
   setSelectedZone: (zone: MapViewZone) => void;
 }
 
-export const AvalancheForecastZonePolygon: React.FunctionComponent<AvalancheForecastZonePolygonProps> = ({
-  zone,
-  setRegionBounds,
-  setSelectedZone,
-}: AvalancheForecastZonePolygonProps) => {
-  const coordinates: LatLng[] = coordinateList(zone.geometry).map(toLatLng);
-  React.useEffect(() => {
-    setRegionBounds((previous: RegionBounds) => updateBoundsToContain(previous, coordinates));
-  }, []); // eslint-disable-line react-hooks/exhaustive-deps
-
+export const AvalancheForecastZonePolygon: React.FunctionComponent<AvalancheForecastZonePolygonProps> = ({zone, setSelectedZone}: AvalancheForecastZonePolygonProps) => {
   return (
     <Polygon
-      coordinates={coordinates}
+      coordinates={toLatLngList(zone.geometry)}
```

Comment:
We could pass in the coordinates we calculated at the owner component, but I figured it would be valuable for this component to always be self-consistent in terms of the state it's using - e.g. no real benefit to allowing a mistake to propagate new coordinates but not a new map zone.